api/internal/git/repospec.go

```diff
@@ -113,67 +110,67 @@ const (
 // From strings like git@github.com:someOrg/someRepo.git or
 // https://github.com/someOrg/someRepo?ref=someHash, extract
 // the parts.
```

Comment:
Minor detail. Can we tweak the `extract the parts` part of this comment? We aren't returning the parts anymore, so it'd be nice if the docstring reflected this. Better yet, we can change it to begin with the function name as well!

Say, something like:
`parseGitURL extracts the parts from strings like git@github.com:someOrg/someRepo.git or https://github.com/someOrg/someRepo?ref=someHash into a RepoSpec`

Reply:
changed the comment based on the latest changes